course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,124 @@
+
+Prerequisites: MAT 136, BIO220H1, EEB225H1, and two of EEB319H1, EEB321H1,
+EEB322H1, EEB323H1
+
+# Time and location
+Tue and Thu 2-4 pm.
+
+Tue, Ramsay Wright (RW 109)
+South west corner of Harbord and St George, 1st floor, enter from St George street.
+
+Thu, Carr Hall (SMC 325)
+Corner of Queen's Park East and St. Joseph Street, 3rd floor, St. Michael's College.
+
+## Computer labs
+Students will learn theoretical ecology concepts and how to program in R. Both lecture halls have access to individual computers for the students. To use the computer workstations, students can login with their UTORid and password. Installed programs and packages are wiped from the computers daily. Saved files will be accessible from any computer via the student's login. Students can use any of the lecture halls when there are no classes scheduled. See the [online schedules](http://lab.chass.utoronto.ca/carr.php) for available times.
+
+# Contacts
+
+## Instructor:
+Prof. Martin Krkosek, martin.krkosek@utoronto.ca, 416-978-3839, RW402
+
+_Please address all general course-related email communication to joel.ostblom@mail.utoronto.ca, and project specific communication to the respective TA of your group. Prefix the subject matter with "EEB430". If you do not receive a reply within 48 hours (excluding week-ends), please send a reminder._
+
+## Teachings assistants: 
+Joel Ostblom (course coordinator), joel.ostblom@mail.utoronto.ca
+Madeleine Bonsma, m.bonsma@mail.utoronto.ca
+Luke Johnston, luke.johnston@mail.utoronto.ca
+Lina Tran, lina.tran@mail.utoronto.ca 
+Elliott Sales de Andrade, esalesde@physics.utoronto.ca
+Lindsay Coome, lindsay.coome@mail.utoronto.ca
+
+_Office hours: Contact us by e-mail to schedule a time._
+
+# Course Web page (on Blackboard)
+All course information will be stored on Blackboard (http://portal.utoronto.ca/). You will have access to the Syllabus, Lecture Schedule & Recommended Readings for each lecture, Lab Schedule, Lab Instructions, Lecture slides (pdf files) and Lab Data sheets. If you have any problem accessing the material, let us know right away so we can fix the problem (e-mail is best).
+
+# Recommended Textbooks
+Different authors take different approaches to the material. Take the time to read different textbooks and to find out which one best suits your learning style.
+
+## Introductory
+Begon M., C.R. Townsend and J.L. Harper.  2006.  Ecology: from individuals to ecosystems.  4th ed. Blackwell Publishing, Malden, MA, USA. QH541.B43 2006 
+Krebs, C.J.  2009.  Ecology: the experimental analysis of distribution and abundance. 6th ed.  Benjamin Cummings, San Francisco.  QH541.28 K74 2001  
+Ricklefs, R.E. and G.L. Miller.  2000.  Ecology.  4th ed.  New York: W.H. Freeman & Co. QH541.R53 2000X. 
+
+## Advanced
+Vandermeer, J.H. and Goldberg, D.E. Population Ecology. Princeton University Press., Princeton, NJ, USA. QH352 .V36 2003X
+
+# Assignments / Course learning outcomes
```

Comment:
I also believe the goals should be tangible and in some ways measurable. Could also have a "Other learning outcomes" section that lists things like team work, report writing, etc.